adam_core/orbits/variants.py

```diff
@@ -104,13 +104,13 @@ def link_to_orbits(
             self,
             left_keys={
                 "orbit_id": orbits.orbit_id,
-                "jd1": orbits.coordinates.time.jd1,
-                "jd2": orbits.coordinates.time.jd2,
+                "day": orbits.coordinates.time.days,
+                "millis": orbits.coordinates.time.millis(),
```

Comment:
This is a consequential choice - what's the linking precision for these variants? I figured milliseconds is good enough, but maybe micros? Maybe seconds? Maybe nanos? You tell me.